pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1713,4 +1719,29 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
     return super.rawListeners(event) as BackendEvents[eventName][];
   }
   // #endregion
+
+  protected async isAdminInstall(): Promise<boolean> {
+    return await new Promise((resolve) => {
+      let key;
+
+      try {
+        key = reg.openKey(reg.HKLM, 'SOFTWARE', reg.Access.READ);
+
+        if (key !== null) {
+          const parsedValue = reg.getValue(key, 'SUSE\\RancherDesktop', 'AdminInstall');
+          const isAdmin = parsedValue !== null && parsedValue === 'true';
```

Comment:
We shouldn't actually care what the value is, if it exists it's an admin install.